packages/bits/spec/virtual-camera/CameraEngine.ts

```diff
@@ -23,10 +23,10 @@ export class CameraEngine {
         if (this.settings.globalLens && LENSES.includes(this.settings.globalLens)) {
```

Comment:
I just noticed that the filename doesn't follow the kebab-case naming convention (`camera-engine.ts`)

Reply:
fixed for all affected files